server/data/restClient.ts

```diff
@@ -23,19 +21,14 @@ interface PostRequest {
   raw?: boolean
 }
 
-interface StreamRequest {
-  path?: string
-  headers?: Record<string, string>
-  errorLogger?: (e: UnsanitisedError) => void
-}
-
 export default class RestClient {
   agent: Agent
 
   constructor(
     private readonly name: string,
     private readonly config: ApiConfig,
-    private readonly token: string = undefined
+    private readonly hmppsToken: string = undefined,
+    private readonly slmToken: string = undefined
```

Comment:
Tried to distinguish between the HMPPS Auth token and Send Legal Mail generated token by their source of origin - they are both "auth" tokens, just different types.